code/script.js

```diff
@@ -1,47 +1,435 @@
+    .getElementById("option2Spoon")
+    .addEventListener("click", option2SpoonClicked);
+};
+
+const option1SpoonClicked = (event) => {
+  event.preventDefault();
+  userAnswers.spoons = option1Spoon.value;
+  console.log(userAnswers.spoons);
+  showMessage(userAnswers.spoons, "user");
+  setTimeout(() => {
+    showMessage("...", "bot");
+  }, 600);
+
+  setTimeout(() => {
+    showMessage(`${userAnswers.spoons} it is!`, "bot");
+  }, 800);
+  setTimeout(() => question5(), 3000);
+};
+
+const option2SpoonClicked = (event) => {
+  event.preventDefault();
+  userAnswers.spoons = option2Spoon.value;
+  console.log(userAnswers.spoons);
+  showMessage(userAnswers.spoons, "user");
+  setTimeout(() => {
+    showMessage("...", "bot");
+  }, 600);
+
+  setTimeout(() => {
+    showMessage(`${userAnswers.spoons} it is!`, "bot");
+  }, 800);
+  setTimeout(() => question5(), 3000);
+};
+
+//Alexander: User is asked to enter their email
+const question5 = () => {
+  showMessage(
+    "Please enter your email, you'll get a notification once it's ready to pick up!",
+    "bot"
+  );
+
+  inputWrapper.innerHTML = `
+  <form id="email">
+  <input id="emailInput" type="email" />
+  <button class="send-btn" type="submit">Send</button>
+  </form>
+      `;
+  document.getElementById("email").addEventListener("submit", handleInput4);
+  //   .addEventListener("click", option1SpoonClicked);
+};
+
+//Alexander: Chatbot prints a confirmation message with the user's name and order
+
+const handleInput4 = (event) => {
+  event.preventDefault(); //prevents refreshing the page
+  userAnswers.email = emailInput.value;
+  emailInput.value = "";
+  showMessage(
+    `Thanks a lot for your order ${userAnswers.name}, your ${userAnswers.order} with ${userAnswers.spoons} of ${userAnswers.toppings} topping is being prepared with love by the Tealicious Lab. See you soon!`,
+    "bot"
+  );
+  setTimeout(
+    () =>
+      (chat.innerHTML = `<h3>Have a good day now!</h3> <video src="assets/BubbleTea Commercial video.mp4" width=80% height="552" muted autoplay loop playsinline></video>`),
+    8000
+  );
+  inputWrapper.innerHTML = `
+  <button id="newOrder" value="">Make a new order! 😍</button>
+  `;
+
+  document.querySelector("#newOrder").addEventListener("click", refresh);
+
+  //userAnswers.order = chatInput.value;
+  // userAnswers.order = teaSelect.value;
+  // console.log(teaSelect.value);
+  // showMessage(`${userAnswers.order}`, "user"); //calling on function
+  // console.log("User chose his drink!"); //can be omitted
+  // showMessage("...", "bot");
+  // setTimeout(() => {
+  //   showMessage("Great choice!", "bot");
+  // }, 1000);
+  // // chatInput.value = ""; //erases the answer field
+  // setTimeout(() => question3(), 2000);
+};
+
+console.log(userAnswers);
+
+// Set up your eventlisteners here
+chatForm.addEventListener("submit", handleInput1);
 // When website loaded, chatbot asks first question.
 // normally we would invoke a function like this:
 // greeting()
```

Comment:
This might be an example of unnecessary comment that could be erased to make your already very clean code even cleaner?